src/library.js

```diff
@@ -498,9 +498,7 @@ LibraryManager.library = {
 #endif
   ],
   emscripten_resize_heap: function(requestedSize) {
-#if CAN_ADDRESS_2GB
-    requestedSize = requestedSize >>> 0;
-#endif
+    requestedSize >>>= 0;
```

Comment:
Please also add a non-emmalloc test for this fix. Just 2 lines to `test_dlmalloc` perhaps?